code/AssetLib/glTF2/glTF2Asset.inl

```diff
@@ -139,6 +139,18 @@ inline CustomExtension ReadExtensions(const char *name, Value &obj) {
     return ret;
 }
 
+inline Extras ReadExtras(Value &obj) {
+    Extras ret;
+
+    ret.mValues.reserve(obj.MemberCount());
+    for (auto it = obj.MemberBegin(); it != obj.MemberEnd(); ++it) {
+        auto &val = it->value;
+        ret.mValues.push_back(ReadExtensions(it->name.GetString(), val));
```

Comment:
Consider using emplace_back instead.

Reply:
Hit the resolve conversation button by accident. Change is added and pushed to branch.